stdlib/filepath/src/test/java/org/ballerinalang/stdlib/filepath/UtilsTest.java

```diff
@@ -34,34 +34,34 @@
     @Test
     public void testGetPathError() {
         InvalidPathException exp = new InvalidPathException("/User/ballerina/path\\test", "Invalid path format");
-        String reason = "INVALID_PATH";
+        //String reason = "INVALID_PATH";
 
         // Get Path error with reason and throwable.
-        ErrorValue error1 = Utils.getPathError(reason, exp);
-        Assert.assertEquals(error1.getReason(), Constants.FILEPATH_ERROR_CODE);
+        ErrorValue error1 = Utils.getPathError(Constants.INVALID_PATH_ERROR, exp.getMessage());
+        Assert.assertEquals(error1.getReason(), Constants.INVALID_PATH_ERROR);
         Assert.assertEquals(error1.getDetails().toString(),
                 "message=Invalid path format: /User/ballerina/path\\test");
 
         // Get Path error without reason.
-        ErrorValue error2 = Utils.getPathError(null, exp);
-        Assert.assertEquals(error2.getReason(), Constants.FILEPATH_ERROR_CODE);
+        ErrorValue error2 = Utils.getPathError(null, exp.getMessage());
+        Assert.assertEquals(error2.getReason(), Constants.GENERIC_ERROR);
         Assert.assertEquals(error2.getDetails().toString(),
                 "message=Invalid path format: /User/ballerina/path\\test");
 
         // Get Path error without throwable.
-        ErrorValue error3 = Utils.getPathError(reason, null);
-        Assert.assertEquals(error3.getReason(), Constants.FILEPATH_ERROR_CODE);
-        Assert.assertEquals(error3.getDetails().toString(), "message=INVALID_PATH");
+        ErrorValue error3 = Utils.getPathError(Constants.INVALID_PATH_ERROR, null);
+        Assert.assertEquals(error3.getReason(), Constants.INVALID_PATH_ERROR);
+        Assert.assertEquals(error3.getDetails().toString(), "message=Unknown Error");
 
         // Get Path error without both reason and throwable.
         ErrorValue error4 = Utils.getPathError(null, null);
-        Assert.assertEquals(error4.getReason(), Constants.FILEPATH_ERROR_CODE);
+        Assert.assertEquals(error4.getReason(), Constants.GENERIC_ERROR);
         Assert.assertEquals(error4.getDetails().toString(), "message=Unknown Error");
 
         // Get Path error without throwable message.
         Exception exp2 = new Exception();
-        ErrorValue error5 = Utils.getPathError(reason, exp2);
-        Assert.assertEquals(error5.getReason(), Constants.FILEPATH_ERROR_CODE);
-        Assert.assertEquals(error5.getDetails().toString(), "message=INVALID_PATH");
+        ErrorValue error5 = Utils.getPathError(Constants.INVALID_PATH_ERROR, exp2.getMessage());
```

Comment:
Shall we have more meaningful variable names instead of  error1, erro2 etc.?

Reply:
The error test-case descriptions are mentioned above as comments